.circleci/config.yml

```diff
@@ -78,8 +78,9 @@ jobs:
             pip install numpy==1.23.5
             pip install -r doc_requirements.txt
             # `asv` pin because of slowdowns reported in gh-15568
-            pip install mpmath gmpy2 "asv==0.4.2" pythran ninja meson click rich-click doit pydevtool pooch
+            pip install mpmath gmpy2 "asv==0.4.2" ninja meson click rich-click doit pydevtool pooch
             pip install pybind11
+            pip install git+https://github.com/serge-sans-paille/pythran.git@feature/backport-xsimd-compatibility-patch
```

Comment:
So, to be sure I understand--`pythran 0.13.0` was released, which fixed the `gast 0.54` issue, but then introduced a separate issue, and now you're pointing to a third (even newer) version of `pythran` on some branch?

Reply:
Yes, I was asked by the phythran maintainer to test what now became 0.13.1, after failing twice to convince him to fix 0.12 first. Thankfully Ralf succeeded where I failed. ;)

I'll rebase this PR if we still need the fix for the tolerances, not yet at my laptop.